e2e/cypress/support/api/plugin.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+// *****************************************************************************
+// Plugins
+// https://api.mattermost.com/#tag/plugins
+// *****************************************************************************
+
+Cypress.Commands.add('apiUploadPlugin', (filename) => {
+    return cy.apiUploadFile('plugin', filename, {url: '/api/v4/plugins', method: 'POST', successStatus: 201});
+});
+
+Cypress.Commands.add('apiEnablePluginById', (pluginId) => {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: `/api/v4/plugins/${encodeURIComponent(pluginId)}/enable`,
+        method: 'POST',
+        timeout: TIMEOUTS.ONE_MIN,
+        failOnStatusCode: true,
+    }).then((response) => {
+        expect(response.status).to.equal(200);
```

Comment:
I've seen you have constants for HTTP status. Maybe used here and in other places too?

Reply:
done.  [`5244344` (#130)](https://github.com/mattermost/mattermost-plugin-demo/pull/130/commits/5244344418883c84aa4a74eba97ea652e6b52b1b)